.taskcluster.yml

```diff
@@ -31,8 +31,8 @@ tasks:
       - taskId: {$eval: as_slugid("lint_test_task")}
         created: {$fromNow: ''}
         deadline: {$fromNow: '1 hour'}
-        provisionerId: aws-provisioner-v1
-        workerType: github-worker
+        provisionerId: proj-relman
```

Comment:
We could change this to `proj-misc` if that project is more appropriate.

Reply:
This project doesn't need secrets yet, but it will (for releases), so proj-relman is better.